litellm/llms/huggingface_restapi.py

```diff
@@ -782,7 +793,10 @@ async def async_streaming(
         model: str,
         timeout: float,
     ):
-        async with httpx.AsyncClient(timeout=timeout) as client:
+        # SSL certificates (a.k.a CA bundle) used to verify the identity of requested hosts.
+        ssl_verify = os.getenv("SSL_VERIFY", litellm.ssl_verify)
```

Comment:
same issue here re: ssl_verify being set to true in the environment (which would be stringified)